### PR TITLE
Automated cherry pick of #2136: Fix misspelling of variant in imageset

### PR DESCRIPTION
**Pull request by rene-dekker**
Cherry pick of #2136 on release-v1.28.

#2136: Fix misspelling of variant in imageset

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Review by caseydavenport — APPROVED** [review]

**Pull request merged**
